src/d/d_camera.cpp

```diff
@@ -45,7 +49,11 @@ namespace {
     }
 
     inline static bool isPlayerGuarding(u32 param_0) {
-        return dComIfGp_checkPlayerStatus1(param_0, 0x80000) ||  daNpc_Md_c::m_mirror;
+        return dComIfGp_checkPlayerStatus1(param_0, 0x80000) || daNpc_Md_c::isMirror();
+    }
+
+    inline static bool isPlayerFlying(u32 param_0) {
+        return dComIfGp_checkPlayerStatus1(param_0, 0x20) || daNpc_Cb1_c::isFlying() || daNpc_Md_c::isFlying();
```

Comment:
```suggestion
        return dComIfGp_checkPlayerStatus1(param_0, daPyStts1_DEKU_LEAF_FLY_e) || daNpc_Cb1_c::isFlying() || daNpc_Md_c::isFlying();
```
Also applies to other PlayerStatus calls in this file (both daPy__PlayerStatus0 and daPy__PlayerStatus1)
And also to check_owner_action/check_owner_action1